app/src/main/res/layout/activity_analysis.xml

```diff
@@ -13,7 +13,7 @@
         android:layout_height="190dp"
         android:layout_alignParentTop="true"
         android:layout_centerHorizontal="true"
-        android:layout_marginTop="46dp"
+        android:layout_marginTop="32dp"
```

Comment:
Also updated the image position in the `activity_analysis.xml` file from 46dp to 32dp marginTop.